codec_test.go

```diff
@@ -22,11 +22,12 @@ func TestCodec_String(t *testing.T) {
 }
 
 func BenchmarkEncoding(b *testing.B) {
+	fmt.Println(benchmarkDivider)
 	// generate some fake data
 	data := generateRandData(128)
 	for _codecType := range codecs {
 		b.Run(
-			fmt.Sprintf("Encoding 128 shares using %s", _codecType),
+			fmt.Sprintf("Encoding 128 shares using %s\n", _codecType),
```

Comment:
Optionally revert that change:
```suggestion
			fmt.Sprintf("Encoding 128 shares using %s", _codecType),
```

The newline doesn't make much sense here as it would just result in an `_` (at least I think this is how go treats whitespaces here).

Reply:
Yes, can confirm. Without the newline:
```
BenchmarkEncoding
BenchmarkEncoding/Encoding_128_shares_using_RSGF8
BenchmarkEncoding/Encoding_128_shares_using_RSGF8-12         	    4750	    237889 ns/op
```
with newline:
```
BenchmarkEncoding
BenchmarkEncoding/Encoding_128_shares_using_RSGF8_
BenchmarkEncoding/Encoding_128_shares_using_RSGF8_-12         	    4092	    250458 ns/op
```

Reply:
Ah that was me playing around with the strings and forgetting to remove the `\n`. Thanks.